src/core/model_config.proto

```diff
@@ -496,6 +496,18 @@ message ModelOptimizationPolicy
     //@@         The value can be one of "FP32", "FP16" and "INT8".
     //@@         Default value is "FP32".
     //@@
+    //@@         "use_calibration" Whether to perform INT8 quantization with
+    //@@         TensorRT calibration. If set to False, quantization nodes are
+    //@@         expected for every tensor in the graph. Default value is true.
+    //@@         This argument is ignored if 'precision_mode' is not 'INT8'.
+    //@@
+    //@@         "is_dynamic_op" Whether at least one input has dynamic shape.
```

Comment:
If we infer this value from the model shape why does it need to be specified explicitly? Perhaps the description is incorrect?

Reply:
The edge case that I am considering is that the model input is dynamic shape but the user specify it as fixed-size tensor in model config. In that case, `is_dynamic_op` should be true but inferring from model config says false.

Reply:
OK to keep it then